packages/accounts/src/light-account/accounts/base.ts

```diff
@@ -222,12 +238,11 @@ export async function createLightAccountBase({
         case "v1.0.2":
           throw new Error(`${type} ${version} doesn't support 1271`);
         case "v1.1.0":
-          return signWith1271Wrapper(hashMessage(message));
+          return signWith1271WrapperV1(hashMessage(message));
         case "v2.0.0":
-          // TODO: implement 1271 signing for v2.0.0
-          throw new Error(
-            `${type} ${version} 1271 signing not implemented yet`
-          );
+          const signature = await signMessageV2({ message });
+          // TODO: handle case where signer is an SCA.
```

Comment:
oof yea, I'm not sure how we can specify this... we can update the `SmartAccountWithSigner` type to include an enum of the signer type to be `EOA` or `SCA` and default to `EOA` in all cases for now which would be backwards compatible